src/settingsview.cpp

```diff
@@ -58,14 +56,27 @@ bool SettingsView::confirmDirDialog(const QString& dir, QString messageText, QSt
     return (ret == QMessageBox::Ok);
 }
 
+bool SettingsView::confirmDialogDownloadDir(QString dir) {
+    auto messageText = gt("download-dir-dialog-msg");
+    messageText = messageText.replace("{{DIRECTORY}}", dir);
+    return confirmDialog(messageText, gt("download-dir-dialog-title"));
+}
```

Comment:
The current ordering of commits is not optimal
1. first `confirmDialogDownloadDir()` is replaced with a more general purpose (yet imperfect) `confirmDirDialog()` (which is later converted to `confirmDialog()`)
2. then `confirmDialogDownloadDir()` is restored as a small wrapper around `confirmDialog()`

A better sequencing is to
1. Extract `confirmDialog()` out of `confirmDialogDownloadDir()` (this is just a preparatory refactoring commit)
2. Use `confirmDialog()` for monitor dir in the next commit

Reply:
> Use confirmDialog() for monitor dir in the next commit

Can this change be included the bigger commit (Add monitor directory entry in Settings panel)?

Reply:
That's what I meant